### DISCUSSION TITLE
Failed to fetch study (id=1)

### DISCUSSION THREAD

**Discussion by jeiglsperger** [optuna/optuna-dashboard#112] [Q&A]
Hi!

I always get the error above and don't know how to solve it. 
In the .db file stands: "Error! /Josef/CTGAN/CTGAN_tryout.db ist not UTF-8 encoded. Saving disabled. See Console for more details."
But the default value for encoding of the storage command should be UTF-8, isn't it?
I use a jupyter notebook for running the code.
Thanks in advance!

**Comment by c-bata** [comment]
As I replied [here](https://github.com/optuna/optuna-dashboard/issues/113), it is difficult to help you without the context and/or steps to reproduce the problem.

**Comment by jeiglsperger** [comment]
Thanks, let's close this duplicate if it's possible. I wasn't sure if the issues or the discussion forum was the right place for my question.
